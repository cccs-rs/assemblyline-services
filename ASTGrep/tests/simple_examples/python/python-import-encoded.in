__import__('\x7a\x6c\x69\x62')
__import__("""\x7a\x6c\x69\x62""")

globals()['\x65\x76\x61\x6c']
globals()[var]
