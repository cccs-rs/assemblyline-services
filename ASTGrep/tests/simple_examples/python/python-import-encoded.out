zlib
zlib

eval
globals()[var]
